src/App.css

```diff
@@ -1,38 +1,22 @@
-.App {
-  text-align: center;
-}
-
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+body {
```

Comment:
- [ ] Kindly consider adding the seperator between the chapter and the progress circle, also try using the same right colors for the little profile icon in the navbar, this will make your app more complete and reflect the original design 

|Your design|Zeplin design|
|-|-|
|![Screenshot_20220625_181951](https://user-images.githubusercontent.com/25227022/175784034-45336c28-a879-4585-aeea-45fd6ab9439f.png)|![Screenshot_20220625_181030](https://user-images.githubusercontent.com/25227022/175783993-7a807aae-884a-49a7-a1b5-e4e76f688958.png)|